src/psyclone/nemo.py

```diff
@@ -344,12 +344,15 @@ def lower_to_language_level(self):
         In-place replacement of this directive concept into language-level
         PSyIR constructs.
 
+        :returns: the lowered version of this node.
+        :rtype: :py:class:`psyclone.psyir.node.Node`
         '''
```

Comment:
Blank line here and below please and copyright year?

Reply:
Done